pastastore/util.py

```diff
@@ -399,3 +400,232 @@ def copy_database(conn1, conn2, libraries: Optional[List[str]] = None,
                 conn2.add_model(mldict, overwrite=overwrite)
         else:
             raise ValueError(f"Library name '{lib}' not recognized!")
+
+
+def frontiers_checks(pstore, check1=True, check1_threshold=0.7,
```

Comment:
I would opt for slightly more descriptive keyword arguments.
check1_rsq, check2_autocorrelation, check3_memory, check4_gain, check5_parambounds?

I would change "path" to the slightly more descriptive "csvdir".

Reply:
I agree.